### PR TITLE
Update Peak actor shader

### PR DESCRIPTION
**Pull request by skoudoro**
in reference of #753, this is the refactoring of the Peaks actor shader to follow our new standard API.

### CODE REVIEW

File: fury/shaders/orient_to_rgb.glsl [hunk 1]
```diff
@@ -1,8 +1,3 @@
-in vec3 center;
-in vec3 diff;
-
-out vec3 centerVertexMCVSOutput;
-
 vec3 orient2rgb(vec3 v)
```
Comment (guaje) [inline fury/shaders/orient_to_rgb.glsl:1]: This file should be in a folder determined by its function. Perhaps, `fury/shaders/utils`.

File: fury/shaders/visible_cross_section.glsl [hunk 2]
```diff
@@ -0,0 +1,7 @@
+bool inVisibleCrossSection(vec3 center)
+{
+    bool xVal = center.x == crossSection.x;
+    bool yVal = center.y == crossSection.y;
+    bool zVal = center.z == crossSection.z;
+    return xVal || yVal || zVal;
+}
```
Comment (guaje) [inline fury/shaders/visible_cross_section.glsl:7]: This file should be in a folder determined by its function. Perhaps, `fury/shaders/interaction`, but I'll leave the name up to you.

File: fury/shaders/visible_range.glsl [hunk 3]
```diff
@@ -0,0 +1,7 @@
+bool inVisibleRange(vec3 center)
+{
+    bool xVal = lowRanges.x <= center.x && center.x <= highRanges.x;
+    bool yVal = lowRanges.y <= center.y && center.y <= highRanges.y;
+    bool zVal = lowRanges.z <= center.z && center.z <= highRanges.z;
+    return xVal && yVal && zVal;
+}
```
Comment (guaje) [inline fury/shaders/visible_range.glsl:7]: This file should be in a folder determined by its function. Perhaps, `fury/shaders/interaction`, but I'll leave the name up to you.

File: fury/actors/peak.py [hunk 4]
```diff
@@ -133,14 +138,49 @@ def __init__(
...
+                if (!inVisibleRange(centerVertexMCVSOutput))
+                    discard;
+            }
+            else
+            {
+                if (!inVisibleCrossSection(centerVertexMCVSOutput))
+                    discard;
+            }
+            """
+
```
Comment (guaje) [inline fury/actors/peak.py:180]: LGTM!

**Review by guaje — CHANGES_REQUESTED** [review]
Hello @skoudoro. Thanks for taking the time to start standardizing the shader API. Here are some of my comments.

**Comment by skoudoro** [comment]
Updated @guaje, Ready for a second review

**Comment by guaje** [comment]
@skoudoro, thanks for this PR. LGTM, merging

**Pull request merged**